File: frontend/src/lib/teams.ts
// Times (setores/supervisões) do Minha Equipe — espelha app/services/performance/teams.py.
// Cada time é um item de menu + uma permissão (árvore do admin) + o slug da rota.

export const GRUPO_PASSIVO = "Contencioso Passivo";
export const GRUPO_CREDITO = "Recuperação de Crédito";

export const TEAMS = [
  { key: "bb-reu", label: "BB Réu", grupo: GRUPO_PASSIVO },
  { key: "bb-execucao", label: "BB Execução & Encerramento", grupo: GRUPO_PASSIVO },
  { key: "bb-acordos", label: "BB Acordos", grupo: GRUPO_PASSIVO },
  { key: "bb-estrategico", label: "BB Estratégico", grupo: GRUPO_PASSIVO },
  { key: "master-reu", label: "Master Réu", grupo: GRUPO_PASSIVO },
  { key: "ativos-reu", label: "Ativos Réu", grupo: GRUPO_PASSIVO },
  { key: "bb-autor-processual", label: "BB Autor — Processual", grupo: GRUPO_CREDITO },
  { key: "ativos-autor", label: "Ativos Autor", grupo: GRUPO_CREDITO },
  { key: "autor-recursal", label: "Autor — Recursal", grupo: GRUPO_CREDITO },
  { key: "ajuizamento", label: "Ajuizamento", grupo: GRUPO_CREDITO },
  { key: "estrategico-autor", label: "Estratégico Autor", grupo: GRUPO_CREDITO },
] as const;

export const TEAM_KEYS = TEAMS.map((t) => t.key);

export function teamLabel(key: string): string {
  return TEAMS.find((t) => t.key === key)?.label ?? key;
}

export function isValidTeam(key: string | undefined): boolean {
  return !!key && TEAM_KEYS.includes(key as (typeof TEAMS)[number]["key"]);
}

File: app/services/performance/teams.py
"""Registro dos times (setores/supervisões) do Minha Equipe.

Cada time é um slug + rótulo. A planilha de squads tem uma aba por subgrupo, e
algumas abas se juntam num setor (ex.: BB Réu = BB Defesa + BB Réu + Recursos);
o mapa aba→setor fica no seed (`TAB_TO_SETOR`). Aqui só a lista canônica usada
pelo gate de permissão, pelos endpoints e pelo menu.
"""

TEAMS = [
    {"key": "bb-reu", "label": "BB Réu"},
    {"key": "bb-execucao", "label": "BB Execução & Encerramento"},
    {"key": "bb-acordos", "label": "BB Acordos"},
    {"key": "bb-estrategico", "label": "BB Estratégico"},
    {"key": "master-reu", "label": "Master Réu"},
    {"key": "ativos-reu", "label": "Ativos Réu"},
    # Autor (Recuperação de Crédito) — 2026-06-29
    {"key": "bb-autor-processual", "label": "BB Autor — Processual"},
    {"key": "ativos-autor", "label": "Ativos Autor"},
    {"key": "autor-recursal", "label": "Autor — Recursal"},
    {"key": "ajuizamento", "label": "Ajuizamento"},
    {"key": "estrategico-autor", "label": "Estratégico Autor"},
]

TEAM_KEYS = {t["key"] for t in TEAMS}
_LABELS = {t["key"]: t["label"] for t in TEAMS}


def team_label(key: str) -> str:
    return _LABELS.get(key, key)


def is_valid_team(key: str) -> bool:
    return key in TEAM_KEYS
